cypress/e2e/DoenetML/tagSpecific/function.cy.js

```diff
@@ -4420,6 +4570,11 @@ describe("Function Tag Tests", function () {
       for (let m of extremumValues) {
         expect(Math.abs(m)).closeTo(1, 0.0001);
       }
+
+      expect(f.stateValues.globalMaximum[1]).eq(1);
```

Comment:
is this a 1-indexed array, should we be asserting what is is in position 0?

Reply:
No.  It is a zero-indexed array.  I'm just not asserting on the locations of the global extrema (the first components) in this case since the option the algorithm chose was arbitrary, and just asserting on their values (the second components).